libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/MathUtils.java

```diff
@@ -0,0 +1,80 @@
+package com.mapbox.services.android.navigation.v5.utils;
+
+
+// TODO Check and remove if not necessary
```

Comment:
If we are using these methods in the nav SDK and they are no longer provided by the libraries we are using, let's add them to our `utils`